packages/process/src/node/process-backend-module.ts

```diff
@@ -40,4 +41,8 @@ export default new ContainerModule(bind => {
             return child.get(TerminalProcess);
         }
     );
+
+    bind(MultiRingBuffer).toSelf().inTransientScope();
+    /* 1MB size, TODO should be a user preference. */
+    bind(MultiRingBufferOptions).toConstantValue({ size: 1048576 });
```

Comment:
Instead of user pref, I think allowing an env would be good enough.

Reply:
Humm I'd rather use a preference.. Is that a problem considering prefs come from the frontend? 
I'll have to tink about it, but I think it can wait a bit until @epatpol PRs are in.